### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Generalized** [hojsgaard/pbkrtest#6]
Dear Authors,

Thank you wholeheartedly for your efforts in bringing Kenward-Roger to R. Currently it supports the lme4 package, but it would be great, if the nlme (gls, lme) was supported too. This would enable the MMRM analysis comparable to SAS. Neither lme4 nor glmmTMB can do that. I noticed, that there is a pull request dated back 2018: "[Adding functions for gls and lme models from nlme package](https://github.com/hojsgaard/pbkrtest/pull/2)". It cannot be merged automatically. Would it be possible for you to look at this, kindly please? Maybe the changes aren't big and the pkbrtest would work with nlme::lme and nlme::gls? nlme is the only package, which allows one to fully manipulate both the random effects and residual covariance structures. Currently, only the Sattherthwaite method is available for it via the emmeans package, but not the Kenward-Roger, which is demanded in various industries, including the pharmaceutical one.

**Comment by NamLQ** [comment]
I agree with this merging. Only the function nlme::lme() can handle the residual covariance structures, but the package *pkbrtest* does not support the lme().
